los_keeper/include/los_keeper/bernstein_polynomial_utils/bernstein_polynomial_utils.h

```diff
@@ -17,28 +20,45 @@ class BernsteinPoly {
     degree_ = -1;
     bernstein_coeff_ = nullptr;
   };
+
   BernsteinPoly(float time_interval[], float bernstein_coeff[], const int &degree);
+
   BernsteinPoly(const BernsteinPoly &bern_poly) {
     time_interval_[0] = bern_poly.time_interval_[0];
     time_interval_[1] = bern_poly.time_interval_[1];
     bernstein_coeff_ = bern_poly.bernstein_coeff_;
     degree_ = bern_poly.degree_;
   };
+
   ~BernsteinPoly();
+
   void SetTimeInterval(float time_interval_[]);
+
   void SetBernsteinCoeff(float bernstein_coeff_[]);
+
   void SetDegree(int degree_);
+
   int GetDegree() const;
+
   bool IsSet() { return ((time_interval_ != nullptr) and (bernstein_coeff_ != nullptr)); };
+
   float *GetTimeInterval() { return time_interval_; };
+
   float *GetBernsteinCoefficient() { return this->bernstein_coeff_; };
+
   float GetValue(float t);
+
   float GetInitialValue() { return bernstein_coeff_[0]; };
+
   float GetTerminalValue() { return bernstein_coeff_[degree_]; };
+
   BernsteinPoly ElevateDegree(int m); // Change to higher degree (m)
   BernsteinPoly operator+(const BernsteinPoly &rhs_);
+
   BernsteinPoly operator-(const BernsteinPoly &rhs_);
+
   BernsteinPoly operator*(const BernsteinPoly &rhs_);
+
```

Comment:
Why did you put new line between functions??

Reply:
I didn't do it on purpose either. Why did it happen like that? I don't know the reason either. I'll remove the redundant line space for you.